ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.hpp

```diff
@@ -125,6 +119,7 @@ sliding_window::ParallelConfig determine_parallel_config(
     uint32_t output_channels,
     const CoreCoord& compute_grid_size,
     ShardOrientation block_shard_orientation,
+    bool enable_channels_padding,
```

Comment:
Let’s avoid adding new bool flags like this. 
https://github.com/tenstorrent/tt-metal/blob/main/contributing/BestPractices.md#14-avoid-bool-arguments-in-apis